alerts/cluster-monitoring-operator/KubeDaemonSetMisScheduled.md

```diff
@@ -0,0 +1,104 @@
+# KubeDaemonSetMisScheduled
+
+## Meaning
+
+The `KubeDaemonSetMisScheduled` alert triggers when, over a defined time interval,
+an excessive number of daemonset pods are found running on nodes where they
+should not be running. This condition typically occurs when a pod is scheduled
+on a node with a taint that doesn't have a corresponding toleration, or when a
+node is unhealthy or overloaded, which can hinder pod scheduling.
+
+## Impact
+
+Daemonset pod may not run on all the expected nodes so some functionalities like
+node metrics or DNS service may be degraded.
```

Comment:
I misread the alert name in my initial review and assumed that it was documenting the KubeDaemonSetRolloutStuck/KubeDaemonSetNotScheduled alerts.

In the case of KubeDaemonSetMisScheduled, the impact is of the daemonset running on more nodes than expected is different, though I'm not sure how we should phrase it. I'm not even sure how to trigger the alert in the first place :thinking:

Reply:
ok, I forgot that we had some background in the original bug :facepalm:

Reply:
My bad I read it incorrectly. I updated with change.